src/components/markdown/ProblemsList/ProblemStatusCheckbox.tsx

```diff
@@ -9,21 +9,28 @@ import UserDataContext from '../../../context/UserDataContext/UserDataContext';
 import Tippy from '@tippyjs/react';
 import 'tippy.js/themes/light.css';
 import styled from 'styled-components';
-
+import feedbackModal from '../../feedbackModal';
 import MarkdownLayoutContext from '../../../context/MarkdownLayoutContext';
 import ConfettiContext from '../../../context/ConfettiContext';
+import Transition from '../../Transition';
+import { logError } from 'gatsby/dist/state-machines/develop/actions';
+import * as firebase from 'firebase';
```

Comment:
try not to import firebase here -- it adds to the bundle size. You can use this instead:

```
const firebase = React.useContext(FirebaseContext);
```